pom.xml

```diff
@@ -8,12 +8,26 @@
     <groupId>twitterapp.src</groupId>
     <artifactId>TwitterApp</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <properties>
+        <dropwizard.version>
```

Comment:
please actually use the properties.  You probably want to update the dropwizard version here too